components/credentials/src/main/java/org/cloudfoundry/credhub/generators/CertificateGenerator.java

```diff
@@ -113,4 +129,13 @@ private boolean shouldUseTransitionalParentToSign(final Boolean allowTransitiona
 
     return transitionalVersionCreatedAt.isAfter(latestNonTransitionalVersionCreatedAt);
   }
+
+  private int getCertificateDuration(final CertificateGenerationParameters params) {
+    if (params.isCa() && caMinimumDuration != null) {
```

Comment:
Is there a reason you didn't make the `*MinimumDuration` variables default to `0` instead of `null`? 

That way, you wouldn't need to check if it's null, you just make the `max` call and it always works.

Reply:
Done.